assignment/tool/src/webapp/js/assignmentsByStudent.js

```diff
@@ -55,21 +57,27 @@ function parseDataCell(html) {
   };
 }
 
-function renderGrouping({ studentName, actionLink, expanded }) {
+function renderGrouping({ studentName, actionLink, expanded, studentUserId }) {
   const template = document.createElement('template');
   template.innerHTML = `
     <tr>
       <td class="border-0">
-        <a href="${actionLink}">
+        <a href="${actionLink}" id="${studentUserId}">
           <span class="expand-icon si ${expanded ? "si-expanded" : "si-collapsed"}"
               aria-hidden="true"></span>
           <span>${studentName}</span>
         </a>
       </td>
     </tr>
   `;
-  
+  if (!itChanged && currentExpandedUserId != "") {
+    changeCurrentExpanded();
+  }
   return template.content;
 }
+function changeCurrentExpanded() {
+  itChanged = true;
+  window.location.href = window.location.href + "#" + currentExpandedUserId;
```

Comment:
Could you run this on this on the load event rather then in here - in the event listener where datatables ins initialized. Did you try using https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView? I think it would be a less error prone and cleaner approach.

Reply:
I try, but if i put it in the window.addEventListener("load") doesn't work